scintillometry/base/base.py

```diff
@@ -0,0 +1,122 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import operator
+import numpy as np
+import astropy.units as u
+from astropy.utils import lazyproperty
+
+
+class ModuleBase(object):
+
+    def __init__(self, ih):
+        self.ih = ih
+        self.offset = 0
+        self._block_index = None
+
+    @lazyproperty
```

Comment:
Why a `lazyproperty` for something so trivial? Actually, I think this should remain up - I don't like relying on private attributes here.

Reply:
I think this is the case for all the lazy properties in base (none require extensive calculation), so changed them all to properties.